src/content/quickstart/openshiftsd/_index.md

```diff
@@ -0,0 +1,166 @@
+---
+date: 2020-02-21T13:36:18+01:00
+title: "OpenShift with Service Discovery (AWS)"
+weight: 15
+---
+
+## AWS
+
+### openshift-install and pull-secret
+
+Download the **openshift-install** and _oc_ tools, and copy your _pull secret_ from:
+
+> https://cloud.redhat.com/openshift/install/aws/installer-provisioned
+
+Find more detailed instructions here:
+
+> https://docs.openshift.com/container-platform/4.3/installing/installing_aws/installing-aws-default.html
+
+
+### Make sure the aws cli is properly installed and configured
+
+Installation instructions
+
+> https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html
+
+```bash
+$ aws configure
+AWS Access Key ID [None]: ....
+AWS Secret Access Key [None]: ....
+Default region name [None]: ....
+Default output format [None]: text
+```
+
+See also for more details:
+
+> https://docs.openshift.com/container-platform/4.3/installing/installing_aws/installing-aws-account.html
+
+### Create and deploy cluster A
+
+In this step you will deploy cluster A, with the default IP CIDRs
+
+| Pod CIDR     | Service CIDR |
+|--------------|--------------|
+|10.128.0.0/14 |172.30.0.0/16 |
+
+
+```bash
+openshift-install create install-config --dir cluster-a
+```
+
+```bash
+openshift-install create cluster --dir cluster-a
+```
+
+The create cluster step will take some time, you can create Cluster B in parallel if you wish.
+
+### Create and deploy cluster B
```

Comment:
This step is only required for testing purposes, it's not a mandatory step for setting up service-discovery. Can you mention that ?